extensions/classifiers/LDAStringClassifier/LDAStringClassifier.py

```diff
@@ -535,6 +537,67 @@ def from_dict(self, model):
         self._c_lw = copy.deepcopy(model['_c_lw'])
         self._c_l = copy.deepcopy(model['_c_l'])
 
+    def _make_json_serializable(self, model):
+        """Makes the StringClassifier dict JSON serializable by
+        converting numpy data types to Python data types.
+
+        Args:
+            model: A dict representing a StringClassifier.
+
+        Returns:
+            model: A dict representing a StringClassifier
+        """
+        model["_b_dl"] = model["_b_dl"].tolist()
+        for k in range(len(model["_b_dl"])):
+            for i in range(len(model["_b_dl"][k])):
+                if isinstance(model["_b_dl"][k][i], numpy.integer):
+                    model["_b_dl"][k][i] = int(model["_b_dl"][k][i])
+                elif isinstance(model["_b_dl"][k][i], numpy.floating):
+                    model["_b_dl"][k][i] = float(model["_b_dl"][k][i])
+        model["_c_dl"] = model["_c_dl"].tolist()
+        for k in range(len(model["_c_dl"])):
+            for i in range(len(model["_c_dl"][k])):
+                if isinstance(model["_c_dl"][k][i], numpy.integer):
```

Comment:
I'm sorry, I don't understand. I converted it from a numpy integer into a normal integer.